test/parallel/test-crypto-random.js

```diff
@@ -462,8 +462,6 @@ assert.throws(
     assert.throws(() => crypto.randomInt(...arg, common.mustNotCall()), {
       code: 'ERR_OUT_OF_RANGE',
       name: 'RangeError',
-      message: 'The value of "max" is out of range. It must be > ' +
```

Comment:
Why not just change the string's wording instead of completely removing the property?

Reply:
Yeah, I already changed my mind and will put that back. But since you ask: The argument for removing it, though, is that since messages can change at any time (unlike the `code` and `name` property), don't test for them so you can conform with "existing test cases don't change for semver-patch changes". But we're pretty far gone from that and I kind of suspect a lot of folks wouldn't consider it an ideal worth pursuing.

Reply:
OK, restored and updated.